packages/ui/src/views/unified-pipeline-studio/components/palette-drawer/components/step-palette-layout.tsx

```diff
@@ -0,0 +1,15 @@
+const StepsPaletteLayout = {
```

Comment:
I doubt if we need so much componentisation for every single div. 
this will not be reused beyond 1 or (maybe) 2 places.